src/modules/shareAlliancePost/assets/util.ts

```diff
@@ -154,14 +154,21 @@ export function createIcon(
     return iconElement;
 }
 
-export function createEditBtn(editBtnClass: string) {
+export function createEditBtn(
+    editBtnClass: string[] | string = [],
```

Comment:
Array string or string as array. Is that necessary?

Reply:
It is "either a array of strings or a string, but by default (if the parameter is not given), it is an empty array)